pyproject.toml

```diff
@@ -9,12 +9,16 @@ version_scheme = "no-guess-dev"
 
 [tool.mypy]
 pretty = true
-show_error_codes = true
 show_error_context = true
```

Comment:
FWIW, I don't like `pretty=true` and `show_error_context=True` very much. Both get in the way of actually processing these results in vscode/pycharm/iterm, or GitHub Actions with a reporting plugin. I disabled them locally, and would be able to commit that part too if you want. The biggest offender is "pretty", which adds a line that doesn't have line numbers, making it really easy to accidentally jump to the top of the file instead of where the violation is.

Reply:
Is there a good alternative that would give a good experience for both working from an ide-like editor (e.g. vscode) and the terminal? These options  are very handy for debugging and working in general in the terminal (that is what I use for workflow)

Reply:
Another idea is we could put the --pretty --show-error-context into tox's default, rather than hard coding it, so that `tox run -e typecheck -- src` would skip it.

Reply:
> Another idea is we could put the `--pretty --show-error-context` into tox's default, rather than hard coding it, so that `tox run -e typecheck -- src` would skip it.

That sounds like a nice compromise... Something like the following?

```
python -m mypy {posargs:--pretty --show-error-context src}
```

Reply:
Implemented.